app/schema/answer.py

```diff
@@ -58,14 +56,12 @@ def check_user_input(user_input):
 
     def get_typed_value(self, raw_input):
 
-        user_input = bleach.clean(raw_input)
```

Comment:
Isn't the fix here not to remove the current bleaching, but to validate all input before storing it? Moving the bleaching to retrieval seems like the wrong way to go, couldn't we have executed something malicious on storing it?